site/docs/develop/java.md

```diff
@@ -36,11 +36,11 @@ import org.projectnessie.client.api.NessieApiV1;
 import org.projectnessie.client.NessieClientBuilder;
 import org.projectnessie.model.Reference;
 
-NessieApiV2 api = NessieClientBuilder.builder()
-  .withUri(URI.create("http://localhost:19121/api/v2"))
-  .build(NessieApiV2.class);
+NessieApiV1 api = NessieClientBuilder.createClientBuilder(null, null)
```

Comment:
API v2 is default now. V1 is deprecated. Does v2 not work?

Reply:
v2 works, but line 27 above says NessieApiV1 should be used. If v2 is the default now, line 27 needs a change then.

Reply:
Pls fix line 27 :)

Reply:
@dimas-b When you say API v2 is default now, I hope this covers all the scenarios, like flink with java, python etc. If so, docs might need to be corrected at few more places. For example - https://projectnessie.org/tools/iceberg/flink/#configuration.

Please confirm.

Reply:
These examples are for direct java client interaction with Nessie Servers (e.g. custom tools). Engines like Spark, Flink, etc. interact with Nessie Servers via `NessieCatalog`, which is part of Iceberg jars. `NessieCatalog` has options for selecting the API version, but that is beyond the scope of this example. IIRC, we need to wait for Iceberg 1.4.0 to release before we can update examples for catalog properties to use v2.

Reply:
Thank you for the explanation. This helps.